lambdas/data-migration2/tests/test-index.js

```diff
@@ -103,7 +103,7 @@ test.after.always(async (t) => {
   await t.context.pdrsModel.deleteTable();
   await t.context.providersModel.deleteTable();
   await t.context.collectionsModel.deleteTable();
-  await t.context.executionsModel.deleteTable();
+  await t.context.executionModel.deleteTable();
```

Comment:
thanks for fixing the plural 😅

Reply:
np, I only noticed it because I was pattern searching for models, I'm sure I've looked at that 1000 times and wasn't even aware of it.